packages/permission-controller/src/PermissionController.ts

```diff
@@ -456,7 +460,7 @@ export type PermissionControllerOptions<
 };
 
 /**
- * The permission controller. See the README for details.
+ * The permission controller. See [the README](../ARCHITECTURE.md) for details.
```

Comment:
Nit: we should resolve this difference in names (i.e. we should call it by the same name as the document)

Reply:
Done here: [b6d186d](https://github.com/MetaMask/controllers/pull/831/commits/b6d186d05b632083b311a04ed895f483a73d4942)